debian/rules

```diff
@@ -14,7 +15,8 @@ override_dh_auto_clean:
 	rm -f aptly
 
 override_dh_auto_build:
-	GOPATH=$(CURDIR) go build -o aptly $(SOURCE_DIR)/main.go
+	cd ${SOURCE_DIR} && go mod download
+	cd ${SOURCE_DIR} && GO111MODULE=on GOBIN=$(CURDIR) go install -v -ldflags "-X main.Version=$(VERSION)"
```

Comment:
Why do you need the `GO111MODULE`?

Reply:
We probably don't. It was there to support older Go.

Reply:
This is now gone.